scalagen/src/main/scala/com/mysema/scalagen/Annotations.scala

```diff
@@ -34,7 +36,7 @@ class Annotations(targetVersion: ScalaVersion) extends UnitTransformerBase {
     
   override def visit(n: AnnotationDecl, arg: CompilationUnit) = {
     // turns annotations into StaticAnnotation subclasses
-    if (targetVersion >= Scala210) {
+    if (targetVersion.sortOrder >= Scala210.sortOrder) {
```

Comment:
This bugged the shit out of me for an hour+,  I can't figure out why the `Ordered` isn't  working here

Reply:
If this is only for compatibility with Scala < 2.10 then we can get rid of it alltogether.